From f801436deca8d63826a89c4f1d98e8cd2d052140 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Sun, 18 Oct 2026 08:24:45 +0200
Subject: [PATCH] Use standard icon names

- mateweather: Replace `stock-unknown` with `image-missing`.
- netspeed: Replace `gtk-dialog-error` with `dialog-error`.
---
 mateweather/data/mateweather-dialog.ui | 2 +-
 mateweather/src/mateweather-applet.c   | 2 +-
 netspeed/src/netspeed.c                | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/mateweather/data/mateweather-dialog.ui b/mateweather/data/mateweather-dialog.ui
index 5beedb05b..319fe041b 100644
--- a/mateweather/data/mateweather-dialog.ui
+++ b/mateweather/data/mateweather-dialog.ui
@@ -404,7 +404,7 @@
                     <property name="visible">True</property>
                     <property name="can_focus">False</property>
                     <property name="valign">start</property>
-                    <property name="icon_name">stock-unknown</property>
+                    <property name="icon_name">image-missing</property>
                     <property name="icon_size">6</property>
                   </object>
                   <packing>
diff --git a/mateweather/src/mateweather-applet.c b/mateweather/src/mateweather-applet.c
index efb2c3081..3ae9ad190 100644
--- a/mateweather/src/mateweather-applet.c
+++ b/mateweather/src/mateweather-applet.c
@@ -482,7 +482,7 @@ update_finish (WeatherInfo *info, gpointer data)
 
 			 icon = weather_info_get_icon_name (gw_applet->mateweather_info);
 			 if (icon == NULL)
-				 icon = "stock-unknown";
+				 icon = "image-missing";
 
 			 n = notify_notification_new (message, detail, icon);
 
diff --git a/netspeed/src/netspeed.c b/netspeed/src/netspeed.c
index 73185abd3..a36adf8a6 100644
--- a/netspeed/src/netspeed.c
+++ b/netspeed/src/netspeed.c
@@ -61,7 +61,7 @@ static const char* wireless_quality_icon [] = {
 
 static const char IN_ICON[] = "go-down";
 static const char OUT_ICON[] = "go-up";
-static const char ERROR_ICON[] = "gtk-dialog-error";
+static const char ERROR_ICON[] = "dialog-error";
 static const char LOGO_ICON[] = "mate-netspeed-applet";
 
 /* How many old in out values do we store?
